dashboard/test/ui/features/star_labs/ailab.feature

```diff
@@ -0,0 +1,66 @@
+Feature: AI Lab
+
+  @no_circle
```

Comment:
@breville and @madelynkasula How do you feel about skipping this test on Drone runs? Currently the test fails because we can't write to the trained models s3 bucket from Drone. @sureshc and I did some initial work to add a CloudFormation template to for the trained-ml-models bucket, but there is additional, more complicated work to sub-divide the current bucket into environment-specific directories. I've started that process, but will need to pass it off before heading out of town and switching teams.  I'd like to have test coverage over the feature in the meantime.  fwiw we [skip Drone on the Oceans UI tests](https://github.com/code-dot-org/code-dot-org/blob/fcae2039f17675fb7ae4b451d2a7c78d304820c1/dashboard/test/ui/features/hour_of_code/ml/ml_hoc.feature#L1).